### DISCUSSION TITLE
Can't find variable: CompressionStream

### DISCUSSION THREAD

**Discussion by akelmj** [honojs/hono#2111] [General]
Why the app crach when i run the server:

I use Bun + Hono 3.12.8

This is the Code I use (index.ts):

`
import { Hono } from 'hono'
import { compress } from 'hono/compress'

const app = new Hono();
app.use('*', compress());
app.get('/', (c) => c.text(`welcome to my APP!`));

export default {
  port: Bun.env.PORT,
  fetch: app.fetch,
}
`

and this the result after i enter to the app url:

![image](https://github.com/honojs/hono/assets/8736967/79421b24-8b27-4ce3-bfb6-ec8e92203cf1)

**Comment by yusukebe** [comment]
As noted on this page, Bun does not support `CompressionStream`.

https://hono.dev/middleware/builtin/compress
